src/components/VMapboxStylePicker.vue

```diff
@@ -41,19 +54,37 @@ export default {
       // initialize control
       this.map.addControl(this, 'bottom-right')
 
+      // creating the variable
+      var year = 2016
+
+      // moment and evetbus connection
```

Comment:
typo: eventbus

Reply:
moment and evetbus connection => moment and eventbus slider connection